src/Features/Core/Portable/NavigateTo/INavigateToSearchCallback.cs

```diff
@@ -10,6 +10,7 @@ namespace Microsoft.CodeAnalysis.NavigateTo
     internal interface INavigateToSearchCallback
     {
         void Done(bool isFullyLoaded);
+        void ReportIncomplete();
```

Comment:
unlike the old nav-to, we can report if results will be inaccurate at any point in the process, not just when we're done.